src/kernel/emulation/linux/ext/epoll_create.c

```diff
@@ -10,7 +10,11 @@ int epoll_create (int __size)
 {
 	int rv;
 
-	rv = LINUX_SYSCALL(__NR_epoll_create, __size);
+	#if __NR_epoll_create
+		rv = LINUX_SYSCALL(__NR_epoll_create, __size);
+	#else
+		rv = LINUX_SYSCALL(__NR_epoll_create1, 0);
```

Comment:
Would it make more sense to call [epoll_create1](https://github.com/darlinghq/darling/blob/master/src/kernel/emulation/linux/ext/epoll_create1.c#L9) instead of writing another `LINUX_SYSCALL `?

Reply:
Eh... I decided to use it anyway